server.js

```diff
@@ -652,6 +652,30 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Flattr integration
+camp.route(/^\/flattr\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var thing = match[1]; // eg, `391332`
+  var format = match[2];
+  var apiUrl = 'https://api.flattr.com/rest/v2/things/' + thing;
```

Comment:
You can flattr either a `thing` or a `URL`. I don't know if you can get the amount of flatters to a URL through the API though.